lxd/api_1.0.go

```diff
@@ -675,22 +707,56 @@ func doAPI10Update(d *Daemon, r *http.Request, req api.ServerPut, patch bool) re
 			}
 		}
 
-		// Validate the storage volumes
-		if nodeValues["storage.backups_volume"] != "" && nodeValues["storage.backups_volume"] != newNodeConfig.StorageBackupsVolume() {
-			// Store validated name back into nodeValues to ensure its not classifed as raw user input.
-			backupsPoolVolume := nodeValues["storage.backups_volume"]
-			nodeValues["storage.backups_volume"], err = daemonStorageValidate(s, backupsPoolVolume)
-			if err != nil {
-				return fmt.Errorf("Failed validation of %q: %w", "storage.backups_volume", err)
+		// Validate the storage volumes.
+		for key := range nodeValues {
```

Comment:
In the interests of reducing cyclomatic complexity would you consider moving this project storage validation logic into a separate function?